service/src/main/resources/HTMLTemplate.html

```diff
@@ -1359,6 +1368,187 @@ <h3 class="heading heading--two">Respondent {{counter}}</h3>
+        {{ respondent.value.party.dateOfBirth | date("d MMMM yyyy", existingFormat="yyyy-MM-dd") }}
+        {% else %}
+        -
+        {% endif %}
+      </dd>
+    </div>
+
+    <div class="data-list__row">
+      <dt class="data-list__title">Gender</dt>
+      <dd class="data-list__definition">
+        {% if respondent.value.party.gender is not empty %}
+        {% if respondent.value.party.gender == 'They identify in another way'
+        and respondent.value.party.genderIdentification is not empty %}
+        <span class="u-block">{{ respondent.value.party.genderIdentification }}</span>
+        {% else %}
+        {{ respondent.value.party.gender }}
+        {% endif %}
+        {% else %}
+        -
+        {% endif %}
+      </dd>
+    </div>
+    <div class="data-list__row">
+      <dt class="data-list__title">Place of birth</dt>
+      <dd class="data-list__definition">
+        {% if respondent.value.party.placeOfBirth is not empty %}
+        {{ respondent.value.party.placeOfBirth }}
+        {% else %}
+        -
+        {% endif %}
+      </dd>
+    </div>
+    <div class="data-list__row">
+      <dt class="data-list__title">Current Address</dt>
+      <dd class="data-list__definition">
+        {% if respondent.value.party.address is not empty %}
+        {% if respondent.value.party.address.AddressLine1 is not empty %}
+        <span class="u-comma-before">{{ respondent.value.party.address.AddressLine1 }}</span>
+        {% endif %}
+        {% if respondent.value.party.address.AddressLine2 is not empty %}
+        <span class="u-comma-before">{{ respondent.value.party.address.AddressLine2 }}</span>
+        {% endif %}
+        {% if respondent.value.party.address.AddressLine3 is not empty %}
+        <span class="u-comma-before">{{ respondent.value.party.address.AddressLine3 }}</span>
+        {% endif %}
+        {% if respondent.value.party.address.PostTown is not empty %}
+        <span class="u-comma-before">{{ respondent.value.party.address.PostTown }}</span>
+        {% endif %}
+        {% if respondent.value.party.address.PostCode is not empty %}
+        <span class="u-comma-before">{{ respondent.value.party.address.PostCode }}</span>
+        {% endif %}
+        {% if respondent.value.party.address.County is not empty %}
+        <span class="u-comma-before">{{ respondent.value.party.address.County }}</span>
+        {% endif %}
+        {% if respondent.value.party.address.Country is not empty %}
+        <span class="u-comma-before">{{ respondent.value.party.address.Country }}</span>
+        {% endif %}
+        {% else %}
+        -
+        {% endif %}
+      </dd>
+    </div>
+    <div class="data-list__row">
+      <dt class="data-list__title">Telephone number</dt>
+      <dd class="data-list__definition">
+        {% if respondent.value.party.telephoneNumber is not empty %}
+        {% if respondent.value.party.telephoneNumber.telephoneNumber is not empty %}
```

Comment:
Please remove inner if statement to the outer if i.e 
{% if respondent.value.party.telephoneNumber is not empty and respondent.value.party.telephoneNumber.telephoneNumber is not empty  %}